ArduCopter/mode_auto.cpp

```diff
@@ -1153,6 +1151,7 @@ void Copter::ModeAuto::do_loiter_unlimited(const AP_Mission::Mission_Command& cm
     }
 
     // start way point navigator and provide it the desired location
+    _mode = Auto_WP;
```

Comment:
shouldn't this be autoloiter ?

Reply:
Probably!

Reply:
nope you are right ... as we need to move to the target first ... we will need to slipt each case as for land or RTL. It is not a big deal but loiter controller is more agressive than wpnav so it can help in some condition